include/mbedtls/ctr_drbg.h

```diff
@@ -45,7 +48,13 @@
 #define MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR                -0x003A  /**< Read or write error in file. */
 
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16 /**< The block size used by the cipher. */
+
+#if !defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY) /* Use default 256-bit key for CTR-DRBG. */
 #define MBEDTLS_CTR_DRBG_KEYSIZE            32 /**< The key size used by the cipher. */
+#else
+#pragma message("Warning: using smaller (128-bit) key size for CTR DRBG may reduce the security of some operations.")
```

Comment:
Since we warn against using 128-bit CTR_DRBG, why are we not officially marking it as `\deprecated`?

Reply:
I'm not sure what is the policy regarding this feature is. Normally when I have seen a feature referred to as sepracated this means that it will soon be removed and users should switch to using something else instead. This might be confusing to put on a new feature/configuration option.